packages/tools/devtools/devtools-view/src/components/data-visualization/EditableValueView.tsx

```diff
@@ -0,0 +1,139 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+
+import { Dropdown, Input, InputOnChangeData, Option, tokens } from "@fluentui/react-components";
+import { EditType, FluidObjectValueNode, SendEditData } from "@fluid-experimental/devtools-core";
+import { useMessageRelay } from "../../MessageRelayContext";
+/**
+ * Input to {@link TreeItem}
+ */
+export interface EditableValueViewProps {
+	node: FluidObjectValueNode;
+	containerKey: string;
+}
+
+/**
+ * Constructs a tree element from the provided header and child contents.
```

Comment:
Looks like this was copy-pasted. We should be sure to update for the editable view context.

Reply:
👍